README.md

```diff
@@ -58,3 +58,22 @@ spec:
 status: { }      
 ```
 
+### Performance
+
+Compile the e2e benchmark suite by:
+
+```shell
+$ make e2e-benchmark-binary
+```
+
+
+The benchmark suite will be creating-updating-deleting configmaps in a flow
+repeatly for 100 times. Here's a comparison of the performance we observed
+in a local experiment:
+
+
+|  Bandwidth  |  Direct          |  ClusterGateway  | ClusterGateway(over Konnectivity) |
+|-------------|------------------|------------------|-----------------------------------|
```

Comment:
is this the latency per request? Direct with 0.6 second seems quite long. Will we expect a linear growth? It might make sense to test based on multiple baselines.

Reply:
no, it's the total latency of three common "write" verbs (1) creating (2) udpating (3) deleting a configmap for the spoke cluster on ACK. basically dividing the latency by 3 will be the per-request.